.vale.ini

```diff
@@ -0,0 +1,11 @@
+# This is the configuration file for the Vale linter.
```

Comment:
@rarkins Can you try running Vale and tell me if you like/hate the output? 

I want to get Vale (nearly) ready for use, before I start moving files into a new repository and worrying about packaging up the Plain Language guidelines.

Also, what's the best way to get vale installed in our Codespace? I've tried getting Vale running in Codespaces but the `brew` package manager or the `gh` CLI program aren't included in our image.

[Vale docs, installing](https://docs.errata.ai/vale/install)

Reply:
is i already wrote, we should add an installer to our Buildpack to install vale when we like it 

until then you can use curl to fetch the binary from release